src/api/support.ts

```diff
@@ -6,6 +6,19 @@ import { pathToFileURL } from 'url'
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { importer } = require('../importer')
 
+function tryRequire(path: string) {
+  try {
+    return require(path)
+  } catch (error) {
+    if (error.code === 'ERR_REQUIRE_ESM') {
+      throw new Error(`Cucumber expected a CommonJS module at '${path}' but found an ES module.
```

Comment:
It might be worth adding the original error via `cause` option so the user gets the full stack e.g. `new Error('...', { cause: error })` as in some cases it could be down to a file that is imported by `path` rather than that one itself, if that makes sense.

Reply:
Oh good spot, hadn't realised that was so new. We'll wait until after dropping Node.js 14 support to use that. I'd still like to include the `message` from the error that we catch as part of our message though, so they can see that path if it is nested (the stack doesn't matter so much).

Reply:
Ok. Done.